direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraDBAccessor.java

```diff
@@ -84,39 +84,57 @@
   @Nullable private transient Cluster cluster;
   /** Session we are connected to. */
   @Nullable private transient Session session;
+  /** Quorum for both reads and writes. */
```

Comment:
Ten komentar nejak nesedi.

Reply:
How come?

Reply:
Quorum is single type of more possible levels so this field can hold also another consistency level.

Reply:
Ah, I think the problem is with the naming of the ConsistencyLevel (QUORUM). Quorum means "the number of people needed to approve a decision". That can be ONE, TWO, MAJORITY (which should be the correct name for QUORUM). So I think that the comment is right, problematic is `ConsistencyLevel.QUORUM` :)